auto_instrument_handler/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "lumigo_auto_instrument",
+  "version": "1.0.0",
+  "description": "Lumigo's special handler for auto-wrapping",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "files": [
+    "index.js"
+  ],
+  "author": "saart",
+  "license": "ISC",
+  "devDependencies": {
+    "jest": "^24.9.0",
+    "@lumigo/tracer": "file:.."
```

Comment:
This may be in dependencies and not dev (the user should upload this file to his func)

Reply:
I don't think so.
He shouldn't upload this package alone - he should just use the tracer, and it's weird that this package will be inside the node_modules of the package *and* in the node_modules of the layer...